ravindu14/SliitSpmProject
# TaskID-#1025 - Added Title to the form interface

How to Test
1.Go to http://localhost:4200/tables url
2. click on download button
3. Check whether there is a title to the given form
